corehq/messaging/smsbackends/telerivet/tasks.py

```diff
@@ -44,3 +46,21 @@ def process_incoming_message(*args, **kwargs):
                 domain_scope=domain_scope, backend_id=backend.couch_id)
         elif kwargs["message_type"] == MESSAGE_TYPE_CALL:
             log_call(from_number, "TELERIVET-%s" % kwargs["message_id"], backend=backend)
+
+
+def process_message_status(sms: SMS, status: str, **kwargs):
+    metadata = {}
+
+    if status == DELIVERED:
+        metadata['gateway_delivered'] = True
+
+    if status in TELERIVET_FAILED_STATUSES:
+        error = kwargs.get('error_message', 'Error occurred')
```

Comment:
nit: i think the fallback should be "No error message" so that it's explicit that there was no message from telerivit.

Reply:
@mkangia Wouldn't "No error message" be a bit ambiguous? 

1) "No error message" as in there's an error but no message
2) "No error message" as in no error, because there's no error message

Or am I reading too deep into it?

Reply:
> am I reading too deep into it?

May be I am. 
The intention for the change is to make it clear that there was no "error message" provided otherwise there is no way to know if the message was from tele or from HQ unless someone looks in the code.
I am not sure if that would confuse someone.
(This is not a blocking comment. Feel free to punt it if it seems it would be confusing)

Reply:
Do we want this translated? if yes we should make this `uggettext_noop('Error occurred')`